src/main/java/com/rocketden/main/service/UserService.java

```diff
@@ -0,0 +1,58 @@
+package com.rocketden.main.service;
+
+import com.rocketden.main.dao.UserRepository;
+import com.rocketden.main.dto.user.CreateUserRequest;
+import com.rocketden.main.dto.user.CreateUserResponse;
+import com.rocketden.main.dto.user.UserMapper;
+import com.rocketden.main.model.User;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class UserService {
+
+    private final UserRepository repository;
+
+    @Autowired
+    public UserService(UserRepository repository) {
+        this.repository = repository;
+    }
+
+    public CreateUserResponse createUser(CreateUserRequest request) {
```

Comment:
> Is this route/method currently used? If not, are we planning on allow people to create actual accounts or require hosts to do so for the MVP?
It is not used beyond testing -- I included this initially because I thought I would first need to create the User object, _then_ add them to the database. However, I was wrong, so I'm happy to remove this -- especially because it would cut down on the relevant code. I don't think we are requiring any actual accounts for the MVP.

> it seems like we might end up with both Users and AnonymousUsers and might need to figure out how to distinguish them
This is a fantastic idea, 100% agree.

Reply:
My personal leaning might be just to leave it as is in case we delete it but end up wanting to pursue it in the near future. However, it can also help to cut down on lines of code, so I'll leave this up to you

Reply:
Ok, I think that's fair, I'll leave this in 👍